src/components/Dropdown/FilterDropdown/FilterDropdown.tsx

```diff
@@ -93,25 +97,58 @@ export const FilterDropdown: FC<Props & ElementProps> = ({
         <DropdownScrollArea>
           <ContentLayout themes={themes}>{children}</ContentLayout>
         </DropdownScrollArea>
-        <BottomLayout themes={themes}>
-          {onReset && (
-            <ResetButtonLayout themes={themes}>
-              <Button variant="text" size="s" prefix={<FaUndoAltIcon />} onClick={onReset}>
-                {resetButton}
-              </Button>
-            </ResetButtonLayout>
+        <ActionArea themes={themes}>
+          <Cluster gap={1} align="center" justify="space-between">
+            {onReset && (
+              <ResetButtonLayout themes={themes}>
+                <Button
+                  variant="text"
+                  size="s"
+                  prefix={<FaUndoAltIcon />}
+                  onClick={onReset}
+                  disabled={isRequestProcessing}
+                >
+                  {resetButton}
+                </Button>
+              </ResetButtonLayout>
+            )}
+
+            <RightButtonLayout>
+              <DropdownCloser>
+                <Button onClick={onCancel} disabled={isRequestProcessing}>
+                  {cancelButton}
+                </Button>
+              </DropdownCloser>
+              <DropdownCloser>
+                <Button variant="primary" onClick={onApply} loading={isRequestProcessing}>
+                  {applyButton}
+                </Button>
+              </DropdownCloser>
+            </RightButtonLayout>
+          </Cluster>
+          {responseMessage && (
+            <>
+              {responseMessage.status === 'success' && (
+                <Message>
+                  <FaCheckCircleIcon
+                    color={themes.color.MAIN}
+                    text={responseMessage.text}
+                    role="alert"
+                  />
+                </Message>
+              )}
+              {responseMessage.status === 'error' && (
+                <Message>
+                  <FaExclamationCircleIcon
+                    color={themes.color.DANGER}
+                    text={responseMessage.text}
+                    role="alert"
+                  />
+                </Message>
+              )}
+            </>
           )}
```

Comment:
nits: この方が見通しよさそうかも?

```suggestion
          {responseMessage?.status === 'success' && (
            <Message>
              <FaCheckCircleIcon
                color={themes.color.MAIN}
                text={responseMessage.text}
                role="alert"
              />
            </Message>
          )}
          {responseMessage?.status === 'error' && (
            <Message>
              <FaExclamationCircleIcon
                color={themes.color.DANGER}
                text={responseMessage.text}
                role="alert"
              />
            </Message>
          )}
```

Reply:
https://github.com/kufu/smarthr-ui/pull/3348/commits/5be554bdbed07c12f812787e43bbc13598081c61
修正しました！